packages/scheduler/crontab

```diff
@@ -0,0 +1,5 @@
+0 0 * * * /usr/scheduler/jobs/runVSExport.sh
+
+0 0 * * * /usr/scheduler/jobs/refreshPerformanceData.sh
+
+0 0 * * * /usr/scheduler/jobs/clearOpenHIMTransactions.sh
```

Comment:
Can all of these write logs to a file please? Check out infrastructure/logrotat.conf

Reply:
https://stackoverflow.com/a/60816190
I think as we're using Alpine if we'd start the cron with `CMD ["crond", "-f", "-l", "2"]` they would go into the container output which get rotated weekly by logrotate.conf